src/platforms/rcore_desktop_rgfw.c

```diff
@@ -1258,6 +1257,16 @@ int InitPlatform(void)
 
     platform.window = RGFW_createWindow(CORE.Window.title, RGFW_RECT(0, 0, CORE.Window.screen.width, CORE.Window.screen.height), flags);
 
+    RGFW_area screenSize = RGFW_getScreenSize();
+    CORE.Window.display.width = screenSize.w;
+    CORE.Window.display.height = screenSize.h;
+    /* 
+        I think this is needed by Raylib now ? 

```

Comment:
Why?

Reply:
I saw it in the glfw platform, I'm not sure if it's an init function Raylib requires or not.